scout/constants/indexes.py

```diff
@@ -62,11 +54,11 @@
         ),
         IndexModel(
             [
+                ("variant_id", ASCENDING),
```

Comment:
And I think we are good with those really; the rest we had was for indexes that we keep around, but that are not in the official constants file. Before we start dropping on stage, some testing is in order so we know it is safe to do on prod with consistent performance.